### DISCUSSION TITLE
Unable to fork: Erroneously saying i already have a fork of it

### DISCUSSION THREAD

**Discussion by NanakoAC** [community/community#23839] [New to GitHub]
<p>Hello everyone, i’m working in a community where lots of people fork from a handful of major parent repos.</p>
<p>The parent repo in question that I’m attempting to fork is…</p>
<p><a href="https://github.com/Baystation12/Baystation12" rel="nofollow noopener">https://github.com/Baystation12/Baystation12</a></p>
<p>However, I am unable to make the fork, i get this error telling me I already have forked it:</p>
<p><img src="https://i.imgur.com/nUGkUi5.png" alt=""></p>
<p>This is not correct. What it’s identifying as a fork of baystation, is actually a fork of this repo:</p>
<p><a href="https://github.com/DS-13-Dev-Team/Dead-Space-13" rel="nofollow noopener">https://github.com/DS-13-Dev-Team/Dead-Space-13</a></p>
<p>THAT repo is forked from baystation, its one i contribute to.</p>
<p>It makes a great many destructive changes that are not suitable for my purpose.</p>
<p>I want to create my own fork of baystation, not this downstream project. How can I do so?</p>

**Comment by seveas** [comment]
<p>Hi <a class="mention" href="/nanakoac">@nanakoac</a>,</p>
<p>I’m afraid you can only have one fork in a repository’s network of forks. If you want to create a fork of the original repository, you will need to either delete your fork of the downstream one or contact support to mark the upstream repo as the upstream of your repository.</p>

**Comment by NanakoAC** [comment]
<p>I don’t understand why though, is that a technical limitation, or a policy of github, or just a UI bug?</p>
<p>As far as I understand, it is possible to work around this using the method described here:</p>
<p><a href="https://handong1587.github.io/linux_study/2015/12/18/create-multi-forks.html" rel="nofollow noopener">https://handong1587.github.io/linux_study/2015/12/18/create-multi-forks.html</a></p>
<p>So i see no technical problem. Is it against the terms of service to do something like that?</p>

**Comment by seveas** [comment, marked as answer]
<p>I’m not sure whether it’s a limitation or a policy, but it’s definitely not a UI bug.</p>
<p>The method you refer to is perfectly fine (both technically and with regards to the terms of service), but does not create multiple forks. The new repo created in step 2 is not registered as a fork in github. This means you cannot create pull requests between this repo and upstream.</p>

**Reply by martin-braun** [reply]
@NanakoAC Please, un-mark this as answer. The only legitimate answer is "We have fixed the issue and it's possible now".

**Reply by Lucbarika100B** [reply]
how did you fix your issue ?

**Comment by NanakoAC** [comment]
<p>I am very unhappy with this outcome, but i guess i can work around it.</p>
<p>Thank you for your help x</p>

**Comment by Sprinterfreak** [comment]
<p>Bump. Don’t know why this is marked solved when it clearly isn’t!<br>
Still the same fundamental issue. GitHub does not support to file PR against multiple (diverged) forks of one repo. Only thing changed seem to be, that there is no error message beeing displayed anymore. A click on “fork” just redirects to the still present, but different personal fork of a diverged repo.</p>
<p>The method referred by nanakoac just creates a totally independend clone, wich can not be used at all to file PR.</p>
<p>Wonder if the only way to get code committed is to</p>
<blockquote>
<p>git sent-email</p>
</blockquote>
<p>to the maintainer like professionals do…</p>

**Comment by martin-braun** [comment]
<p>You have to be kidding. Please involve your work mates now, because this is a limitation that makes no sense and is rather destructive. I want to contribute to several crypto coins that all fork from bitcoin/bitcoin, but your website forbids me to. What kind of … is this? Sorry for sounding harsh, it’s obviously not your fault, but it’s such a terrible limitation. How am I supposed to contribute with PRs to multiple forks of the same network?</p>

**Comment by thiswallz** [comment]
terrible lack of features here.

**Reply by martin-braun** [reply]
Yes, and GitHub staff seems to be ignorant about this issue, or we would at least get a statement of acknowledgement from them. They lure us with free storage and software, but lack to innovate further in our interest, while feeding our code for their AI products that will eventually compete with us. Self-hosting your own infrastructure was never that important like it is today.

**Comment by mm12** [comment]
Gee, I wonder if thus will ever be ACTUALLY fixed.

**Comment by Awesomerly** [comment]
Insert passive-aggressive comment here

**Comment by InvictusNavarchus** [comment]
Can't believe this still hasn't been fixed after all these years.
